rfcs/0013-synthea-future-claims-and-automation.md

```diff
@@ -0,0 +1,150 @@
+
+### Background on Synthea
+
+Mitre Synthea:
+
+* [Synthea](https://github.com/synthetichealth/synthea) -- for generating synthetic beneficiary and claim data to load into BFD DB.
+  * How To Differentiate Synthea And Real Data:
+    - Synthetic beneficiary IDs are a negative number.
+    - Synthetic MBIs have the character 'S' in a certain position.
+
+Generating & Loading Synthea Data: 
+- BFD clones the master branch of the Synthea codebase, and generates the RIF files via the `run_Synthea` command, where the batch size and geography can be set, as well as a Synthea.properties file, which is important, for defining numeric ranges of key claim and beneficiary properties i.e. beneficiary ID, claim group ID, claim date, etc.
+- The RIF files are copied into the BFD Server codebase, and loaded into the local database. 
+- If there are no errors loading locally, the RIF files are uploaded to AWS S3 and loaded into TEST, PROD SBX, PROD databses via the same ETL process with CCW RIF files. 
+- Synthetic data is now available to BFD users.
+
+* In addition to these steps, BFD follows a thorough test plan before generating or loading data into TEST, PROD SBX, or PROD environments. 
+  * [Synthea Test Plan](https://github.com/CMSgov/beneficiary-fhir-data/blob/master/apps/bfd-model/bfd-model-rif-samples/dev/synthea-test-plan.md) 
+
+Motivation:
+As previous synthetic data releases were separated by 3 months, and only 20,000 synthetic beneficiaries released, data became quickly outdated, and the amount of data being released doesn't justify the usecase of synthetic data. Additionally, there is inefficiency involved with producing and releasing synthetic data when done without automation. Manual process are prone to human error, and can cause incidents, such as accidental overlap of beneficiary IDs between different synthetic data batches. There are more than enough reasons to look into streamlining and scaling up synthetic data production to maximize the benefit to users and partners. This involves automated recurring Synthea data generation, automated recurring loading, large Synthea data generation, and potential required changes to the BFD server and pipeline application.
+
+## Proposed Solution
+[Proposed Solution]: #proposed-solution
+
+Automated Recurring Generation: 
+To increase the benefit and quality of synthetic data to the user, data that is current, or for future use, needs to be generated on a more regular basis. The proposed plan is weekly and quarterly.
+
+  - Weekly - Synthetic data will be generated with a set batch size using the master branch of the Synthea codebase in a hosted cloud instance, along with the Synthea end-state properties file from the most recent generated batch, which is hosted in AWS S3. The end-state properties file is critical for making sure the next batch of Synthea data will not overlap with released Synthea data.
```

Comment:
I think we decided in the last Synthea meeting that we would only generate quarterly although we may have future claims data loaded weekly

Reply:
This is correct. Fixing this